foxmlToSolr.xslt

```diff
@@ -289,6 +291,17 @@
         </xsl:for-each>
       </xsl:if>
 
+      <xsl:if test="string($index_parent_models) = 'true'">
+        <xsl:variable name="parent_models">
+          <xsl:call-template name="get-parent-models">
+            <xsl:with-param name="PID" select="$PID" />
+          </xsl:call-template>
+        </xsl:variable>
+
+        <xsl:for-each select="xalan:nodeset($parent_models)//sparql:model[@uri != concat('info:fedora/', $PID)]">
```

Comment:
Not sure the predicate (the `[@uri != concat('info:fedora/', $PID)]` bit) makes sense here... the PID should never occur as a model, should it? Even if it did... would we not want to output it?